libs/Microsoft.MixedReality.WebRTC/PeerConnection.cs

```diff
@@ -280,11 +280,201 @@ public struct VideoCaptureFormat
         public uint fourcc;
     }
 
+    /// <summary>
+    /// Settings to create a new transceiver wrapper.
+    /// </summary>
+    public class TransceiverInitSettings
+    {
+        /// <summary>
+        /// Transceiver name, for logging and debugging.
+        /// </summary>
+        public string Name;
+
+        /// <summary>
+        /// Initial value of <see cref="Transceiver.DesiredDirection"/>.
+        /// </summary>
+        public Transceiver.Direction InitialDesiredDirection = Transceiver.Direction.SendReceive;
+
+        /// <summary>
+        /// List of stream IDs to associate the transceiver with.
+        /// </summary>
+        public List<string> StreamIDs;
+    }
+
+    /// <summary>
+    /// Wrapper for an event possibly delayed.
+    /// </summary>
+    public class DelayedEvent
```

Comment:
nit: possibly move the class to a separate file. Also make internal unless you want to add it to the public API/have people take a dependency on it.

Reply:
Moving to `internal` and leaving there, it's only used by `PeerConnection`.

Reply:
Fixed.